.publicApi/Microsoft.ApplicationInsights.dll/net452/PublicAPI.Shipped.txt

```diff
@@ -880,6 +880,7 @@ Microsoft.ApplicationInsights.TelemetryClient.Initialize(Microsoft.ApplicationIn
 Microsoft.ApplicationInsights.TelemetryClient.InitializeInstrumentationKey(Microsoft.ApplicationInsights.Channel.ITelemetry telemetry) -> void
 Microsoft.ApplicationInsights.TelemetryClient.InstrumentationKey.get -> string
 Microsoft.ApplicationInsights.TelemetryClient.InstrumentationKey.set -> void
+Microsoft.ApplicationInsights.TelemetryClient.TelemetryConfiguration.get -> Microsoft.ApplicationInsights.Extensibility.TelemetryConfiguration
```

Comment:
Please add this entry to `PublicAPI.UnShipped.txt`. This file needs an update to all target framework.

Reply:
Got it. Sorry for the inconvenience. For my understanding, is there any way to open those file from within Visual Studio? It seems to always point to the 452 version of the file in the solution explorer, even when changing the target framework.

Reply:
Using Visual Studio, you can select to fix all occurances of an issue in the Project/Solution.
This will fix it in all frameworks.
Screenshots how to here: https://github.com/dotnet/roslyn-analyzers/issues/3322

Reply:
@Expecho As I commented earlier, please move API definition from `PublicAPI.Shipped.txt` to `.publicApi/Microsoft.ApplicationInsights.dll/net452/PublicAPI.Unshipped.txt`. This line `Microsoft.ApplicationInsights.TelemetryClient.TelemetryConfiguration.get -> Microsoft.ApplicationInsights.Extensibility.TelemetryConfiguration` should move to `PublicAPI.Unshipped.txt` for all target framework.

Reply:
You're right sorry. And after reading https://www.natmarchand.fr/prevent-breaking-changes-apis-roslyn/ I do have a better understanding of the way these files work.